pulp-sdk build failed

Hi,

I'm building the most recent patch, a1d73765373792a741123432e02defa84dec02bd, but the build fails as below.

Steps followed to build pulp-sdk:
export PULP_RISCV_GCC_TOOLCHAIN= <set to the right build directory>
source configs/pulp.sh

Build error:
install -D PULP/pulp-sdk/platform/vp/models/build/models/utils/injector_impl.so PULP/pulp-sdk/pkg/sdk/dev/install/ws/python/utils/injector_impl.so
make[2]: *** No rule to make target 'PULP/pulp-sdk/platform/vp/models/build/models/utils/composite_impl/utils/composite_impl.o', needed by 'PULP/pulp-sdk/platform/vp/models/build/models/utils/composite_impl.so'.  Stop.
make[2]: Leaving directory 'PULP/pulp-sdk/platform/vp/models'
Makefile:9: recipe for target 'build' failed
make[1]: *** [build] Error 2
make[1]: Leaving directory 'PULP/pulp-sdk/platform/vp'
Reached EOF with exit status 2
Makefile:4: recipe for target 'all' failed
make: *** [all] Error 255

Is the makefile broken in this latest patch? I was able to build the previous patch, 79daee01e5fc21b6d833c68e866a148e98f11f90.

Thank you!

--- a/README.md
+++ b/README.md
@@ -23,7 +23,7 @@ Be carefull that this procedure has been validated with a default installation o
 Starting from a fresh Ubuntu 16.04 distribution, here are the commands to be executed to get all required dependencies:
 
     $ sudo apt install git python3-pip python-pip gawk texinfo libgmp-dev libmpfr-dev libmpc-dev swig3.0 libjpeg-dev lsb-core doxygen python-sphinx sox graphicsmagick-libmagick-dev-compat libsdl2-dev libswitch-perl libftdi1-dev cmake scons libsndfile1-dev
-    $ sudo pip3 install artifactory twisted prettytable sqlalchemy pyelftools openpyxl xlsxwriter pyyaml numpy configparser pyvcd
+    $ sudo pip3 install artifactory twisted prettytable sqlalchemy pyelftools 'openpyxl==2.6.4' xlsxwriter pyyaml numpy configparser pyvcd
     $ sudo pip2 install configparser
 
 In case you face a problem regarding python dependencies you may also want to check if the version reported by python3 is matching the python version reported by pip:

